File: lua/refactoring/tests/refactor/106/lua/return/extract.expected.lua
-- stylua: ignore start

local function foo_bar(a, test)
    local test_other = 11
    for idx = test - 1, test_other do
        print(idx, a)
    end
    return test_other
end


function simple_function(a)
    local test = 1
    local test_other = foo_bar(a, test)


    return test, test_other
end

File: lua/refactoring/tests/refactor/106/php/class/extract.expected.php
<?php

class RefuctorPlease
{
    public function foo_bar (
        $a, $test
    ) {
        $test_other = 11;
        for ($idx = $test - 1; $idx < $test_other; ++$idx) {
            print $idx;
            print $a;
        }
        return $test_other;
    }

    public function multiple_returns($a)
    {
        $test = 1;
        $test_other = $this->foo_bar($a, $test);


        return $test_other;
    }
}
